src/error.rs

```diff
@@ -19,6 +21,66 @@ pub enum ProtocolError {
     UnsupportedCompressionFlag(u8),
 }
 
+impl<T> fmt::Display for Error<T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match *self {
+            Error::Grpc(ref _status, ref _header_map) =>
+                write!(f, "gRPC error"),
```

Comment:
Take it or leave it: When formatting a string literal with no format arguments, you could also just call `f.pad("gRPC error")` (and so on).